### ISSUE TITLE
Just a question

### ISSUE THREAD

**Issue by sjmcdowall** [mikowals/pause-publish#1]
I need something exactly like this -- I have a Meteor app that gets data real-time in an IoT sort of way from a device that generates a ton of test data each second or so.  I'd like to pause the publication while the data is updated and restart it.  So this seems perfect. Only question, is this still a viable package and the right way to do it or with Meteor 1.2. is there a better way??  Thanks!

**Comment by mikowals** [comment]
I think the problem this was trying to solve might have gone away when Meteor core made oplog tailing more efficient (https://github.com/meteor/meteor/issues/4694).  My understanding of that fix, is that it pauses the observers in a smarter way by detecting when they get behind.  Though I guess if you have big db changes that happen periodically maybe pausing the publish functions manually still makes sense.

I don't think there have been any breaking changes to this package but haven't tried it in months.  I will run the test app again sometime in the next few days and see what happens.

**Comment by sjmcdowall** [comment]
Yeah — every 2 seconds or so once a test starts — there will be a crap load of data generated and “updated” into the DB .. 3000-5000 records per update block .. 

Did the tests still work? :) 

/Steve
